extract/omie_extractor.py

```
@@ -0,0 +1,417 @@
+            
+        Returns:
+            None
+        """
+        # Validate input dates
+        fecha_inicio_carga, fecha_fin_carga = self.fecha_input_validation(fecha_inicio_carga, fecha_fin_carga)
+
+        # Convert to datetime objects
+        fecha_inicio_carga_dt = datetime.strptime(fecha_inicio_carga, '%Y-%m-%d')
+        fecha_fin_carga_dt = datetime.strptime(fecha_fin_carga, '%Y-%m-%d')
+
+        print(f"Processing market: continuo")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)